server/src/main/java/org/elasticsearch/health/node/ShardsCapacityHealthIndicatorService.java

```diff
@@ -27,18 +29,24 @@
 import org.elasticsearch.indices.ShardLimitValidator;
 
 import java.util.ArrayList;
+import java.util.Iterator;
 import java.util.LinkedHashSet;
 import java.util.List;
+import java.util.Locale;
+import java.util.Map;
 
 /**
  *  This indicator reports health data about the shard capacity across the cluster.
  *
- * <p>
+
```

Comment:
This formatting looks a bit off with an empty line in the middle of a JavaDoc. Could you revert the `<p>` line?